Ballerina.toml

```diff
@@ -1,4 +1,3 @@
 [project]
-org-name = "wso2"
+org-name = "wso2-guides"
```

Comment:
We don't allow hyphens in the org-name. We only allow lowercase alphanumerics and underscores for the org-name. What is the reason for changing this?

Reply:
The reason for the change is
https://github.com/ballerina-platform/ballerina-lang/issues/11275

Reply:
Fixed with https://github.com/ballerina-guides/salesforce-twilio-integration/pull/20/commits/9f07f83c403f22990a50db03ffb03f46f9badc8d